coreos-installer

```diff
@@ -378,6 +378,43 @@ write_networking_opts() {
     echo "set ignition_network_kcmdline=\"$(cat /tmp/networking_opts)\"" >> /mnt/boot_partition/ignition.firstboot
 }
 
+#########################################################
+# Helper to persist ignition platform id to initramfs first boot
+#########################################################
+write_platform_id() {
+    local tmp_platform_id="/tmp/platform_id"
+    local proc_cmdline="/proc/cmdline"
+
+    if [ ! -f ${tmp_platform_id} ]; then
+        log "Not overwriting ignition platform id, no platform id provided"
+        return
+    else
+        local platform_id="$(cat ${tmp_platform_id})"
+    fi
+
+
+    log "Overwriting ignition platform id"
+    # check for the boot partition
+    mkdir -p /mnt/boot_partition
+    mount_boot_partition /mnt/boot_partition
+    trap 'umount /mnt/boot_partition; trap - RETURN' RETURN
+    local conf_files=$(ls /mnt/boot_partition/loader/entries/ | grep 'conf')
+
+    for f in $conf_files
+    do
+        # Update boot loader entry first
+        local conf_files_path="/mnt/boot_partition/loader/entries/$f"
+        if grep -q 'ignition.platform.id' $conf_files_path; then
+            sed -i "s/ignition.platform.id=.* /ignition.platform.id=${platform_id} /" $conf_files_path
+        else
+            echo " ignition.platform.id=${platform_id}" >> $conf_files_path
+        fi
+    done
+
+    rm -f ${tmp_platform_id}
+    log "Set ignition.platform.id=\"${platform_id}\""
```

Comment:
maybe add `kernel CLI parameter in BLS configs` to the end of that log statement.

Reply:
Sure, updating :)